specs/ledger/hs/src/Ledger/Update.hs

```diff
@@ -424,7 +424,7 @@ instance STS UPVOTE where
             ) <- judgmentContext
         vts' <- trans @ADDVOTE $ TRC ((rups, dms), vts, vote)
         let pid = vote ^. vPropId
-        Core.psSize (Set.singleton pid ◃ vts') < (pps ^. cfmThd) || pid `Set.member` (dom cps) ?! LowerThanThdAndNotAlreadyConfirmed
+        Core.psSize (Set.singleton pid ◃ vts') < (pps ^. cfmThd) || pid `Set.member` (dom cps) ?! HigherThanThdAndAlreadyConfirmed
```

Comment:
If the name of the condition is the negation of the guard (i.e. what comes before the "oh-noes" operator `?!`) shouldn't this be `HigherThanThdAndNotConfirmed`?

Reply:
Whoops, yes. I negated both parts of the previous name, rather than the condition 🙂